include/openthread/ip6.h

```diff
@@ -871,6 +871,10 @@ typedef struct otBorderRoutingCounters
     otPacketsAndBytes mInboundMulticast;  ///< The counters for inbound multicast.
     otPacketsAndBytes mOutboundUnicast;   ///< The counters for outbound unicast.
     otPacketsAndBytes mOutboundMulticast; ///< The counters for outbound multicast.
+    uint32_t          mRaRx;              ///< The number of received RA packets.
+    uint32_t          mRaTx;              ///< The number of transmitted RA packets.
+    uint32_t          mRsRx;              ///< The number of received RS packets.
+    uint32_t          mRsTx;              ///< The number of transmitted RS packets.
```

Comment:
I think we need to initialize these variables, or else it initial value depends on the usage, which we may want to avoid this behaviour.

Reply:
Good catch! I just added the reset step in the constructor.

Reply:
Great, will you add it in this PR, or the following up PRs ?

Reply:
Sorry I didn't make it clear. I've added the clear step in the constructor of `Ip6`.